### PR TITLE
Remove incorrect example for "attach data rule" tutorial 

### PR DESCRIPTION
**Pull request by yangyansong-adbe**

### CODE REVIEW

**Review by calebk1 — APPROVED** [review]
LGTM!

File: resources/user-guides/attach-data.md [hunk 1]
```diff
@@ -153,13 +153,7 @@ If you do not have existing rules for this property, the **Create New Rule** but
 4. On the right pane, in the **JSON Payload** field, type the data that will be added to this event.
 5. Click **Keep Changes**.
 
-On the right pane, you can add a freeform JSON payload that adds data to an SDK event before an extension that is listening for this event can hear the event. In this example, a custom mbox with custom mbox parameters is added to the event before the Target extension processes it. The added custom mbox will now be added on outgoing Target retrieve location requests.
```
[the remark this answers is not in the document]
Comment (swarna04) [reply resources/user-guides/attach-data.md:156]: The edits here look fine to me.

**Comment by swarna04** [comment]
@yangyansong-adbe Thank you for fixing this and apologies it slipped under my radar. Can you please resolve the review comments and merge conflicts here.

**Comment by yangyansong-adbe** [comment]
@swarna04  Conflicts are merged, let me know if I can merge this PR.

**Comment by swarna04** [comment]
Thanks @yangyansong-adbe , can you please resolve Emilia's comment as well before merging the pr.

**Review by swarna04 — APPROVED** [review]

**Comment by yangyansong-adbe** [comment]
@swarna04 I have addressed that in an other PR. https://aep-sdks.gitbook.io/docs/release-notes#ios-aepuserprofile-3-0-1

**Pull request merged**